grape-entity.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
   s.rubyforge_project = 'grape-entity'
 
   s.add_runtime_dependency 'multi_json', '>= 1.3.2'
-  s.add_runtime_dependency 'activesupport', '>= 5.0.0'
+  s.add_runtime_dependency 'activesupport'
```

Comment:
please change instead to: `> 4`, else it would fail this spec`./spec/grape_entity/hash_spec.rb:6 # Grape::Entity except option for nested entity`) with `< 4`